build.sbt

```diff
@@ -1,34 +1,190 @@
-import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import uk.gov.hmrc.DefaultBuildSettings.{ defaultSettings, integrationTestSettings, scalaSettings }
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import com.lucidchart.sbt.scalafmt.ScalafmtCorePlugin.autoImport._
+import uk.gov.hmrc.ExternalService
+import uk.gov.hmrc.SbtBobbyPlugin.BobbyKeys.bobbyRulesURL
+import uk.gov.hmrc.ServiceManagerPlugin.Keys.itDependenciesList
+import play.twirl.sbt.Import.TwirlKeys
+import play.sbt.routes.RoutesKeys
 
 val appName = "secure-message-frontend"
 
 val silencerVersion = "1.7.0"
 
+lazy val externalServices = List(
+  ExternalService("DATASTREAM")
+)
+
+lazy val wartremoverSettings =
+  Seq(
+    WartRemoverSettings.wartRemoverError,
+//    wartremoverErrors in (Test, compile) -= Wart.NonUnitStatements, // does not seem to work as intended
```

Comment:
Maybe suppress the specific errors if you think there is good reason as described here http://www.wartremover.org/doc/install-setup.html - rather than turning off completely?

Reply:
all test assertions fail on that Wart.NonUnitStatements, so was trying to to suppress all of them just for the tests.